### DISCUSSION TITLE
Test runner angular

### DISCUSSION THREAD

**Discussion by vvs09** [storybookjs/storybook#25478] [Help]
Does test runner work with angular? Enable to setup test runner with an angular project.when I try to run npm run test-storybook I get an error could not load main.js in .storybook. I tried the flag --confi-dir but says unknown command test-storybook

**Comment by Marklb** [comment]
I use the test-runner in my Angular projects. I haven't looked into how the test runner is working exactly, but I have briefly skimmed the code a while ago and don't think the framework being used should affect it.

Can you provide your "test-storybook" command. If you can provide a repro then that would be best, but the command may be enough to see how you are using the test-runner.

**Reply by vvs09** [reply]
Thank you for your response, it worked for me.But is installing playwright a dependency for test runner to work as I got an error for not having playwright (npx playwright install )?

**Reply by Marklb** [reply]
I don't know too much about playwright, but I think it stores the browsers in a user data directory that is outside of your project and used by all playwright instances on that machine. When there are browser updates, it seems to require the install command to be run. I normally just run the command it gives me. I haven't looked into why it requires the command to be run manually and doesn't just prompt if I want to let it update them, but there may be something that explains it in the playwright docs.

**Reply by vvs09** [reply]
Thank you!
